### PR TITLE
Wording and refactoring of docs

### PR DESCRIPTION
**Pull request by leodido**
**What type of PR is this?**

/kind cleanup

/kind documentation

**Any specific area of the project related to this PR?**

/area build

/area driver-kmod

/area driver-ebpf

/area libscap

/area libsinsp

**What this PR does / why we need it**:

It just refactors and adjusts the wording of comments and documentation comments around the project.

**Which issue(s) this PR fixes**:

NONE

**Special notes for your reviewer**:

No API changes have been made.

No changes to variable names have been made.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by leogr — APPROVED** [review]
/approve

**Review by leogr — APPROVED** [review]

File: userspace/libscap/scap_savefile.c [hunk 1]
```diff
@@ -2526,7 +2526,7 @@ int32_t scap_read_init(scap_t *handle, gzFile f)
 	if(sh.major_version > CURRENT_MAJOR_VERSION)
 	{
 		snprintf(handle->m_lasterr, SCAP_LASTERR_SIZE,
-			 "cannot correctly parse the capture. Upgrade your version of sysdig.");
+			 "cannot correctly parse the capture. Upgrade your version.");
```
Comment (gnosek) [inline userspace/libscap/scap_savefile.c:2529]: At this point the second sentence could just be "Upgrade."

**Review by gnosek — APPROVED** [review]

**Pull request merged**